backend/battling/models.py

```diff
@@ -4,10 +4,13 @@
 from users.models import User
 
 
+STATUS_CHOICES = [("ONGOING", "Ongoing"), ("SETTLED", "Settled")]
```

Comment:
Django now supports a better choices: https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types

Reply:
I changed it, but I don't think I quite understand how it works :/